### PR TITLE
Feature/sentieon genomics

### PR DESCRIPTION
**Pull request by Miles-Perry**

### CODE REVIEW

File: var/spack/repos/builtin/packages/sentieon-genomics/package.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    For instructions on how to set up a mirror, see
+    http://spack.readthedocs.io/en/latest/mirrors.html"""
+
+    homepage = "https://www.sentieon.com/"
+    url      = "file://{0}/sentieon-genomics-201808.01.tar.gz".format(os.getcwd())
+
+    version('201808.01', sha256='6d77bcd5a35539549b28eccae07b19a3b353d027720536e68f46dcf4b980d5f7')
+
+    # Licensing.
+    license_require = True
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/sentieon-genomics/package.py]:
```suggestion
    license_required = True
```

File: var/spack/repos/builtin/packages/sentieon-genomics/package.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    license_url     = 'https://support.sentieon.com/manual/_downloads/Sentieon.pdf'
+
+    def configure(self, spec, prefix):
+        config = {
+            'destinationFolder':   prefix,
+            'licensePath':         self.global_license_file
+        }
+
+    def install(self, spec, prefix):
+        self.configure(spec, prefix)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/sentieon-genomics/package.py]: This line calls your `configure` function above, correct? This function doesn't actually do anything.

File: var/spack/repos/builtin/packages/sentieon-genomics/package.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        install_tree('bin', prefix.bin)
+        install_tree('doc', prefix.doc)
+        install_tree('etc', prefix.etc)
+        install_tree('lib', prefix.lib)
+        install_tree('libexec', prefix.libexec)
+        install_tree('share', prefix.share)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/sentieon-genomics/package.py]: Not sure if there are garbage files in the download or not, but if you just want to copy everything, you can do:
```python
install_tree('.', prefix)
```

**Comment by alalazo** [comment]
Duplicate of #9557

**Pull request closed without merging**